src/Schema/Directives/Fields/CreateDirective.php

```diff
@@ -33,14 +46,21 @@ public function resolveField(FieldValue $fieldValue): FieldValue
             function ($root, array $args) {
                 $modelClassName = $this->getModelClass();
                 /** @var Model $model */
-                $model = new $modelClassName;
+                $model = new $modelClassName();
 
                 $flatten = $this->directiveArgValue('flatten', false);
                 $args = $flatten
                     ? reset($args)
                     : $args;
 
-                return MutationExecutor::executeCreate($model, collect($args));
+                if (! $this->directiveArgValue('transaction',
+                    config('lighthouse.transactional_mutations', true))) {
```

Comment:
Let's remove this option.

Reply:
I will wait for an hour or so for consensus from everyone